RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -1269,3 +1324,29 @@ double AntiElectronIDMVA6::dCrackEta(double eta)
 
   return std::abs(retVal);
 }
+
+bool AntiElectronIDMVA6::atECalEntrance(const reco::Candidate* part, math::XYZPoint &pos)
+{
+  bool result = false;
+  double bField = 3.8; // hopefully Teslas are proper units (better to take it from event setup) 
```

Comment:
indeed, it would be better to pick this up from the event setup.
```c++
      edm::ESHandle<MagneticField> pSetup;
      es.get<IdealMagneticFieldRecord>().get(pSetup);
      auto bField = pSetup->inTesla(GlobalPoint(0,0,0)).z()
```